polynote-server/src/test/scala/polynote/server/KernelPublisherIntegrationTest.scala

```diff
@@ -83,9 +83,13 @@ class KernelPublisherIntegrationTest extends FreeSpec with Matchers with ExtConf
       val process         = kernel.transport.asInstanceOf[SocketTransportServer].process
 
       val collectStatus = ZStream.fromHub(kernelPublisher.status)
-        .takeUntil(_ == KernelBusyState(false, false))
-        .runCollect.map(_.toList)
-        .forkDaemon.runIO()
+        .mapAccum((false, false)) {
```

Comment:
This change is unrelated and should have gone in #1201. Will try to move it there.